Sources/Extensions/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  DispatchQueueExtensions.swift
+//  SwifterSwift
+//
+//  Created by Quentin Jin on 2018/10/13.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(Dispatch)
+import Dispatch
+
+// MARK: - Properties
+public extension DispatchQueue {
+
+    /// SwifterSwift: A Boolean value indicating whether the current
+    /// dispatch queue is the main queue.
+    public static var isMainQueue: Bool {
+        enum Static {
```

Comment:
Can this be marked as `private`? If so you should probably add it.

Reply:
No, it can't be `private`, will get a compile error